src/charts/mapd-table.js

```diff
@@ -291,6 +291,10 @@ export default function mapdTable (parent, chartGroup) {
         .html(d => formatDataValue(d[col.name]))
         .classed("filtered", col.expression in _filteredColumns)
         .on("click", d => {
+          // detect if user is selecting text or clicking a value, if so don't filter data
+          const s = window.getSelection().toString()
+          if (s.length) { return }
```

Comment:
I tend to favor condensing statements, so putting `window.getSelection().toString()` directly in the if condition, as more variables = more to remember. Just a personal preference though.

Reply:
Makes sense. I considered making this a one liner, but thought that making it more verbose would be slightly more readable for future devs.